src/environment/player/action_accuse.py

```diff
@@ -0,0 +1,30 @@
+from environment.player_action import PlayerAction
+from environment.game_state import GameState
+from environment.player.action_result import ActionResult
+from typing import List
+from random import shuffle
+
+class ActionAccuse(PlayerAction):
+
+    def get_suspects(self, game_state: GameState) -> List[str]:
+        suspects = [suspect['name'] for suspect in game_state.suspects.values()]
+        shuffle(suspects)
+        return suspects
+    
+    def get_killer(self, game_state: GameState) -> str:
+        return game_state.killer['name']
```

Comment:
1. Zaujima ma, preco si sa rozhodol pouzit metody pre vratenie podozrivych a vraha z herneho stavu?

2. `get_suspects` a `get_killer` podla toho ako ich pouzivas, nie su metody, ktore patria do public API `ActionAccuse`. navrhujem prefixnut pomocou jedneho underscore. je to konvencia v Pythone pre private.